R/explanation.R

```diff
@@ -332,3 +336,21 @@ get_list_approaches <- function(n_features, approach) {
   }
   return(l)
 }
+
+#' @keywords internal
+explainer_x_test <- function(x_test, feature_labels) {
+
+  if (ncol(x_test) == length(feature_labels)) {
+
+    if (identical(colnames(x_test), feature_labels))
+      return(as.matrix(x_test))
+  }
```

Comment:
Suggests to delete all these lines and always go through the data.table conversion. Doesn't take much time and ensures identical $x_test output from explain (otherwise you rownames will be different)

Reply:
Thanks, you just explained to me why the tests failed when doing the data.table conversion :-)

Reply:
Fixed with cbc6e5e.